bindings/pyroot/src/RootWrapper.cxx

```diff
@@ -316,6 +317,18 @@ static int BuildScopeProxyDict( Cppyy::TCppScope_t scope, PyObject* pyclass ) {
    if ( ! isNamespace && ! hasConstructor )
       cache[ "__init__" ].push_back( new TConstructorHolder( scope, (Cppyy::TCppMethod_t)0 ) );
 
+   // Add function templates that have not been instantiated to the class dictionary
+   auto cppClass = TClass::GetClass(Cppyy::GetFinalName(scope).c_str());
+   TIter next(cppClass->GetListOfFunctionTemplates());
+   TFunctionTemplate *templ = nullptr;
+   while ((templ = (TFunctionTemplate*)next())) {
```

Comment:
```
for (auto templ: TRangeStaticCast<TFunctionTemplate>(cppClass->GetListOfFunctionTemplates())) {
```
is strictly equivalent but sooo much nicer :-)

Reply:
Sure, thanks for the tip!